test/src/test/java/jenkins/model/JenkinsBuildsAndWorkspacesDirectoriesTest.java

```diff
@@ -67,6 +70,29 @@ private void clearSystemProperties() {
                 .forEach(System::clearProperty);
     }
 
+    @Issue("JENKINS-53284")
+    @Test
+    public void changeWorkspacesDirLog() throws Exception {
+        loggerRule.record(Jenkins.class, Level.WARNING)
+                .record(Jenkins.class, Level.INFO).capture(1000);
+
+        story.then(step -> {
+            assertFalse(logWasFound(LOG_WHEN_CHANGING_WORKSPACES_DIR));
+            setWorkspacesDirProperty("testdir1");
+        });
```

Comment:
Shouldn't you test the log is found at level INFO?

Reply:
At that stage, we have already started (i.e. **without** any flag), and about to restart), so in that first case: **no** log at all is expected, so this way works fine.

As you can see below, on the second restart after having set the property: then yes we must check the level more specifically.

Does that clarify @MRamonLeon?

Reply:
Yes, I was asking to check when the message is raised in INFO level.